doc/source/advanced.rst

```diff
@@ -260,6 +272,23 @@ Passing a list of labels or tuples works similar to reindexing:
 
    df.loc[[('bar', 'two'), ('qux', 'one')]]
 
+.. warning::
+
+   It is important to note that tuples and lists are not treated identically
+   in pandas. Whereas a tuple is interpreted as one multi-level key, a list is
```

Comment:
in pandas "when it comes to multi-indexing" ? to make clear that in many other places we don't make this distinction (or is that already clear from the context?)

Reply:
I would say "when it comes to indexing", since ``pd.Series(range(3)).loc[(1,2)]`` doesn't work (and I think this is good).

Reply:
Done